### DISCUSSION TITLE
Questions about new setting?

### DISCUSSION THREAD

**Discussion by wu520609** [andreynovikov/trekarta#269] [General]
Hello: There is 2 new setting in new version.

1.About "route color" There is a New setting for router color setting.
But why There is two router color setting? What is the difference between "Router color" and "Active router color"

2.About new setting "to prohibit map rotation".  where is this setting,I cant find it.

**Comment by SokoFromNZ** [comment]
I haven't tried 1 yet.

But for 2 it took me a little too :)
You have to rotate the map so the compass icon is shown. Then long-press on the compass-icon and it smaps to north and stays like this.

**Comment by andreynovikov** [comment]
1. When you navigate to point or via route a line is drawn from your current position to destination - this is "active route". Some day it'll become true route.
